scripts/TPI.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import scipy.optimize as opt
+
+
```

Comment:
Should we have this while loop inside a function, like `solve_TP` or something so we can call it from the execute file?

Reply:
yes, go ahead and make the change please.

Reply:
I would recommend you do it because we do not have access to the code before approval of pull request unless copy and paste your code in a new file. I think you can do it quicker

Reply:
@saharnazb Sure, will do all ASAP